### ISSUE TITLE
moderation: delete posts

### ISSUE THREAD

**Issue by macjohnny** [mittwald/typo3-forum#166]
when deleting a single post of a topic, the topic is deleted instead.

**Comment by macjohnny** [comment]
sorry, this was due to incorrect postCount, which resulted from deleting posts manually in the backend. 

@martin-helmich how about a hook for updating postCount when deleting posts in the backend?

**Comment by martin-helmich** [comment]
>how about a hook for updating postCount when deleting posts in the backend?

Yeah, these counter columns are prone to inconsistencies. The count should either be guaranteed to update when sth. is changed in the backend, or we should outright prohibit editing/deleting posts in the backend (which would probably suck). @pstranghoener, what's your take on this?

**Comment by macjohnny** [comment]
@martin-helmich @pstranghoener ping
